crates/bounded_planet_core/src/networking/systems.rs

```diff
@@ -173,103 +181,107 @@ pub async fn handle_connection(
                 )).expect("Failed to send network event");
                 break;
             },
-            Ok((send, recv)) => {
-                tokio::spawn(handle_stream(guid, send, recv, event_sender.clone()));
+            Ok(recv) => {
+                tokio::spawn(handle_incoming_stream(id, recv, event_sender.clone()));
             }
         };
     }
 
     // Send a final event indicating that this connection closed
     event_sender
-        .send(ReceiveEvent::Disconnected(guid))
+        .send(ReceiveEvent::Disconnected(id))
         .expect("Failed to send network event");
 }
 
 /// Handle all the work of a specific stream
-async fn handle_stream(
+async fn handle_incoming_stream(
```

Comment:
This is a bit of a nitpick, but even just reading this review, the similarity between the name of this function and the above one (`handle_incoming_stream` vs `handle_incoming_streams`) tripped me up. Might be better to differ the names a little for readability